sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -81,7 +81,7 @@ public partial class AzureDeveloperCliCredentialOptions : Azure.Identity.TokenCr
     public partial class AzurePipelinesCredential : Azure.Core.TokenCredential
     {
         protected AzurePipelinesCredential() { }
-        public AzurePipelinesCredential(string tenantId, string clientId, string serviceConnectionId, Azure.Identity.AzurePipelinesCredentialOptions options = null) { }
+        public AzurePipelinesCredential(string systemAccessToken, string clientId = null, string tenantId = null, string serviceConnectionId = null, Azure.Identity.AzurePipelinesCredentialOptions options = null) { }
```

Comment:
This doesn't seem to match the implementation. Is one of them outdated?

Reply:
yes - meant to re-export.